src/devices/Ads1115/Ads1115.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    /// Analog-to-Digital Converter ADS1115
+    /// </summary>
+    public class Ads1115 : IDisposable
+    {
+        private I2cDevice _sensor = null;
+
+        private readonly byte _inputMultiplexer;
+        private readonly byte _measuringRange;
+        private readonly byte _dataRate;
+
+        /// <summary>
+        /// Initialize a new Ads1115 device connected through I2C
+        /// </summary>
+        /// <param name="sensor">I2C Device, like UnixI2cDevice or Windows10I2cDevice</param>
+        /// <param name="inputMultiplexer">Input Multiplexer</param>
+        /// <param name="measuringRange">Programmable Gain Amplifier</param>
+        /// <param name="dataRate">Data Rate</param>
+        public Ads1115(I2cDevice sensor, InputMultiplexer inputMultiplexer = InputMultiplexer.AIN0, MeasuringRange measuringRange = MeasuringRange.FS4096, DataRate dataRate = DataRate.SPS128)
+        {
+            _sensor = sensor;
+            _inputMultiplexer = (byte)inputMultiplexer;
+            _measuringRange = (byte)measuringRange;
+            _dataRate = (byte)dataRate;
+
+            Initialize();
+        }
+
+        /// <summary>
+        /// Initialize ADS1115
+        /// </summary>
+        private void Initialize()
+        {
+            // Details in Datasheet P18
+            byte configHi = (byte)((_inputMultiplexer << 4) +
```

Comment:
double checking: `+` or `|`?

Reply:
I tested every sensor on the Raspberry Pi 3B+ running Raspbian😁